src/widgets/modals/SearchForm.js

```diff
@@ -0,0 +1,159 @@
+/* eslint-disable react/forbid-prop-types */
+/**
+ * mSupply Mobile
+ * Sustainable Solutions (NZ) Ltd. 2020
+ */
+
+import React, { useMemo, useState } from 'react';
+import { View, StyleSheet, FlatList, Text } from 'react-native';
+import PropTypes from 'prop-types';
+
+import { FormControl } from '../FormControl';
+
+import { getFormInputConfig } from '../../utilities/formInputConfigs';
+
+import { APP_FONT_FAMILY, DARK_GREY } from '../../globalStyles';
+import { TouchableOpacity } from 'react-native-gesture-handler';
+import { queryPatientApi, queryPrescriberApi } from '../../utilities/network/lookupApi';
+
+const RECORD_TYPES = {
+    PATIENT: 'patient',
+    PRESCRIBER: 'prescriber',
+}
+
+const FORM_CONFIGS = {
+    [RECORD_TYPES.PATIENT]: 'searchPatient',
+    [RECORD_TYPES.PRESCRIBER]: 'searchPrescriber',
+}
+
+const LIST_CONFIGS = {
+    [RECORD_TYPES.PATIENT]: [ 'firstName', 'lastName', 'dateOfBirth' ],
+    [RECORD_TYPES.PRESCRIBER]: [ 'firstName', 'lastName', 'registrationCode' ],
+}
+
+const SearchListItemColumn = ({ value, type }) => (
+    <View style={localStyles.columnContainer}>
+        <Text style={localStyles.text}>
+            {type === 'date' ? value.toDateString() : value}
+        </Text>
+    </View>
+);
+
+const SearchListItem = ({ listItem, listConfig }) => {
```

Comment:
I think it might've been a better idea to adapt the current `ResultRow` component which is used in the `AutoCompleteSelector` (even though that would need a refactor and would be more work)

Reply:
I did consider that, but came to the conclusion that you said re. it being a bit more work at this stage. Kind of still trying to do the minimum changes to existing code just to get a demo working. Once everything works, will definitely revisit suggestions for refactors like this!